snap/info.go

```diff
@@ -851,10 +851,7 @@ type AppInfo struct {
 	RefreshMode     string
 	StopMode        StopModeType
 
-	// TODO: this should go away once we have more plumbing and can change
-	// things vs refactor
-	// https://github.com/snapcore/snapd/pull/794#discussion_r58688496
-	BusName string
+	ActivatesOn []string
```

Comment:
this should be close to the definition of Sockets I think, also I would kind of expect ti to be a ActivatesOn []*SlotInfo